bokeh/properties.py

```diff
@@ -508,11 +515,11 @@ class Int(PrimitiveProperty):
 
 class Float(PrimitiveProperty):
     """ Floating point type property. """
-    _underlying_type = (float, ) + bokeh_integer_types
+    _underlying_type = numbers.Real,
```

Comment:
That was the plan.

I just realized that I hadn't made these tuples previously and that it would probably cause CI to fail. So, this was a quick way to hack it in and repush. I just want to see if the general strategy works first before I get sucked into the nitty gritty details.